Character/Classes/Druid/Druid (Dreams).xml

```diff
@@ -1,47 +1,46 @@
+				<text>	The sphere vanishes at the end of the rest or when you leave the sphere.</text>
+			</feature>
 		</autolevel>
 
 		<autolevel level="10">
-            <feature optional="YES">
-                <name>Circle of Dreams: Hidden Paths</name>
-				<text>Starting at 10th level, you can use the hidden, magical pathways that some fey use to traverse space in the blink of an eye. As a bonus action on your turn, you can teleport up to 60 feet to an unoccupied space you can see. Alternatively, you can use your action to teleport one willing creature you touch up to 30 feet to an unoccupied space you can see.</text>
-				<text>You can use this feature a number of times equal to your Wisdom modifier (minimum of once), and you regain all expended uses of it when you finish a long rest.</text>
-            </feature>
+			<feature optional="YES">
+				<name>Circle of Dreams: Hidden Paths</name>
+				<text>Starting at 10th level, you can use the hidden, magical pathways that some fey use to traverse space in a blink of an eye. As a bonus action on your turn, you can teleport up to 60 feet to an unoccupied space you can see. Alternatively, you can use your action to teleport one willing creature you touch up to 30 feet to an unoccupied space you can see.</text>
+				<text>	You can use this feature a number of times equal to your Wisdom modifier (minimum of once), and you regain all expended uses of it when you finish a long rest.</text>
+			</feature>
 		</autolevel>
 
 		<autolevel level="14">
-            <feature optional="YES">
+			<feature optional="YES">
 				<name>Circle of Dreams: Walker in Dreams</name>
-				<text>When you finish a short rest, you can cast one of the following spells, without expending a spell slot or requiring material components: dream (with you as the messenger), scrying, or teleportation circle.</text>
-				<text>This use of teleportation circle is special. Rather than opening a portal to a permanent teleportation circle, it opens a portal to the last location where you finished a long rest on your current plane of existence. If you haven*t taken a long rest on your current plane, the spell fails but isn't wasted.</text>
+				<text>At 14th level, the magic of the Feywild grants you the ability to travel mentally or physically through dreamlands.</text>
+				<text>When you finish a short rest, you can cast one of the following spells, without expending a spell slot or requiring material components: Dream (with you as the messenger), Scrying, or Teleportation Circle.</text>
+				<text>This use of Teleportation Circle is special. Rather than opening a portal to a permanent teleportation circle, it opens a portal to the last location where you finished a long rest on your current plane of existence. If you haven't taken a long rest on your current plane, the spell fails but isn't wasted.</text>
```

Comment:
```suggestion
				<text>	This use of Teleportation Circle is special. Rather than opening a portal to a permanent teleportation circle, it opens a portal to the last location where you finished a long rest on your current plane of existence. If you haven't taken a long rest on your current plane, the spell fails but isn't wasted.</text>
```